### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by imran5644**
I create an app that looks for information on the Weather and shows the temperature, weather description.
Use async, await, and promises
Fetch cities from a JSON file and search for cities that begin with the search

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,14 @@
   },
   "homepage": "https://github.com/imran5644/weatherApp#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```
Comment (MahmoudBakr23) [inline package.json:23]: Please add the following scripts:-
First is the dev script which is building the app in development mode

- [x] ```"dev": "webpack --mode development"```

Then the build script which is building the app in production mode (for deployment)

- [x] ```"build": "webpack --mode production"```

Although, you could do ```"build": "webpack"``` for both modes at the same time, but recommended separated.

<hr>

And the optional script is: 

- ```"start": "webpack serve"``` to start a live server for the app

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello there :)

## **Highlights ✔✔**

You've done excellent work building this app. I liked the changing images based on the weather data returned, also you did a fantastic job keeping the logic code separated from the DOM manipulation code, and your Linter, PR summary, and commit messages are well-done!
However, there is always room for improvement. Please consider my comments below:-

## **To Improve 👌**

- [x] Please include any necessary data bout how to find, generate and install the API keys for non-technical future viewers. The current instructions aren't enough.
- [x] In your package.JSON file, kindly add the missed scripts that needed to run and build the app. See also my inline comments below.
scripts_requirement

## **OPTIONAL, but nice to have 😊🙏**

- You did a great job separating the code, but I see that you choose to keep the switching units button in the call-api file. You could easily keep it in the DOM manipulation file since it is one.
- In the scripts of the JSON file, it'd be great if you added a script to start a live server for the app.
```"start":"webpack serve"```
Also, see it in my inline comments down below.

<hr>

That's it, please do the required changes, then ask for another CRR. 👍 
Good luck and happy coding 😊🙏
Please feel free to contact me on Slack with the name "Mahmoud Bakr" if you got any questions!

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Imran Yousaf** 
Good job with the improvements for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
Just one last detail to improve before you can go ahead:

- [x] In your package.json file, add the next:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  }
```
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello Imran Yousaf** 
Amazing job :tada: :tada: :tada:  you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the develop branch
 
Keep going like this  :clap: :clap:
 
Oscar Andrés Russi Porras

**Pull request merged**